spm_funcs.py

```diff
@@ -54,8 +54,12 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    img = nib.load(fname)
+    fdata = img.get_fdata()
+    ret_vals = []
+    for i in range(fdata.shape[-1]):
+        ret_vals.append(spm_global(fdata[...,i]))
+    return ret_vals
```

Comment:
A small point - notice that the docstring says the values should be an array, but you currently have a list.  It might be good to convert the values to an array here.